Client/Frontend/Browser/Tabs/InactiveTabCell.swift

```diff
@@ -129,18 +129,14 @@ extension InactiveTabCell: UITableViewDataSource, UITableViewDelegate {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         switch InactiveTabSection(rawValue: indexPath.section) {
         case .inactive:
-            let cell = tableView.dequeueReusableCell(withIdentifier: OneLineTableViewCell.cellIdentifier, for: indexPath) as! OneLineTableViewCell
-            cell.customization = .inactiveCell
-            cell.backgroundColor = .clear
-            cell.accessoryView = nil
-            cell.bottomSeparatorView.isHidden = false
+            let cell = tableView.dequeueReusableCell(withIdentifier: InactiveTabItemCell.cellIdentifier, for: indexPath) as! InactiveTabItemCell
```

Comment:
Could be a guard here with `as?` and return a simple UITableViewCell in case of dequeuing error?